ccdproc/combiner.py

```diff
@@ -177,6 +177,61 @@ def scaling(self, value):
             # reshape so that broadcasting occurs properly
             self._scaling = self.scaling[:, np.newaxis, np.newaxis]
 
+    # set up IRAF-like minmax clipping
+    def iraf_minmax_clipping(self, nlow=None, nhigh=None):
+        """Mask pixels using an IRAF-like minmax clipping algorithm.  The
+        algorithm will mask the lowest nlow values and the highest nhigh values
+        before combining the vlues to make up a single pixel in the reuslting
+        image.  For example, the image will be a combination of
+        Nimages-low-nhigh pixel values instead of the combination of Nimages.
+
+        Note that this differs slightly from the nominal IRAF behavior when
+        other masks are in use.  For example, if the nhigh>=1 and the highest
+        pixel is already masked for some other reason, then this algorithm will
+        count the masking of that pixel toward the count of nhigh masked pixels.
+        IRAF behaves slightly diifferently in this case (see IRAF help for that
+        behavior): http://stsdas.stsci.edu/cgi-bin/gethelp.cgi?imcombine
+        
+        Here is a copy of the relevant IRAF help text:
+        
+        nlow = 1, nhigh = (minmax)
+            The number of low and high pixels to be rejected by the "minmax"
+            algorithm. These numbers are converted to fractions of the total
+            number of input images so that if no rejections have taken place
+            the specified number of pixels are rejected while if pixels have
+            been rejected by masking, thresholding, or nonoverlap, then the
+            fraction of the remaining pixels, truncated to an integer, is used.
+        
+        Parameters
+        -----------
+        nlow : int or None, optional
+            If not None, the number of low values to reject from the
+            combination.
+            Default is ``None``.
+
+        nhigh : int or None, optional
+            If not None, the number of high values to reject from the
+            combination.
+            Default is ``None``.
+        """
+        self.data_arr.sort(axis=0)
+        nimages = self.data_arr.mask.shape[0]
+        newmask = np.ndarray(self.data_arr.mask.shape, dtype=bool)
+
+        if nlow != None:
+            if nlow > nimages:
```

Comment:
Maybe better to check nlow and nhigh together? Currently one could discard 10 nlow pixel and 10 nhigh pixel with 10 stacked images.

Is the ValueError meant as Warning or is it necessary because it would fail otherwise?